src/pages/elements/pictograms/overview.mdx

```diff
@@ -3,10 +3,625 @@ title: Pictograms
+
+<Row>
+<Column colLg={12}>
+
+![Angles example](images/07_Angles_1.svg)
+
+</Column>
+</Row>
+
+<Row>
+<Column colLg={12}>
+
+![Angles example](images/07_Angles_2.svg)
+
+</Column>
+</Row>
+
+<Row>
+<Column colMd={4} colLg={4}>
+<DoDontExample type="do" caption="Do use multiple of 15° or an angle that best represents the metaphor when necessary.">
+
+![](images/07_Angles_3_do.svg)
+
+</DoDontExample>
+</Column>
+<Column colMd={4} colLg={4}>
+<DoDontExample type="dont" caption="Don't use 45° angles exclusively for all icons. It won't work.">
+
+![](images/07_Angles_4_dont.svg)
+
+</DoDontExample>
+</Column> 
+</Row> 
+
+### Details 
+
+Pictograms are more illustrative than UI icons but should not be overly detailed. Communicate your ideas with only the most essential elements. Avoid using perspective and unnecessary visual metaphors.
+
+<Row>
+<Column colMd={4} colLg={4}>
+<DoDontExample type="do" caption="Do use thoughtful metaphors. Create depth through flat layers and essential details.">
+
+![](images/08_Details_1_do.svg)
+
+</DoDontExample>
+</Column>
+</Row>
+
+<Row>
+<Column colMd={4} colLg={4}>
+<DoDontExample type="dont" caption="Don't use cliches or perspective views. Small details will not scale down well.">
+
+![](images/08_Details_2_dont.svg)
+
+</DoDontExample>
+</Column>
+<Column colMd={4} colLg={4}>
+<DoDontExample type="dont" caption="Don't combine pictograms or elements unless contributing a new pictogram">
+
+![](images/08_Details_3_dont.svg)
+
+</DoDontExample>
+</Column> 
+</Row> 
+<Row>
+<Column colMd={4} colLg={4}>
+<DoDontExample type="dont" caption="Don't duplicate or manipulate artwork to create effects.">
+
+![](images/08_Details_4_dont.svg)
+
+</DoDontExample>
+</Column>
+<Column colMd={4} colLg={4}>
+<DoDontExample type="dont" caption="Don't scale other pictograms to combine or use different stroke weights">
+
+![](images/08_Details_5_dont.svg)
+
+</DoDontExample>
+</Column>
+</Row>
+
+## Color
+
+Pictograms are illustrative and can be used at larger scales, therefore a wider variety of visual styles are acceptable. They are by default a solid, monochromatic color but may be treated in four distinct styles: black or white, monochromatic color, tinted or shaded color, and gradient. Regardless of style, pictograms need to pass the same color contrast ratio as typography (4.5:1). For more information on color, see [IBM Design Color Usage](https://www.ibm.com/design/language/elements/color/).
+
+<Row>
+<Column colLg={12}>
+
+![](images/09_Color_1.svg)
+
+<Caption>
+The grid system and guidance allows for diverse solutions.
+</Caption>  
+</Column>
+</Row>
+
+<Row>
+<Column colLg={12}>
+
+![](images/09_Color_2.svg)
+
+<Caption>
+Pictograms can be treated in four distinct styles.
+</Caption> 
+</Column>
+</Row>
+<Row>
+
+<Column colLg={12}>
+
+![](images/09_Color_3.svg)
+
+<Caption>
+Use background color values of 70-100 to achieve luminosity.
+</Caption> 
+</Column>
+</Row>
+
+<Column colMd={4} colLg={4} noGutterSm>
```

Comment:
This needs to get wrapped in a `<Row>` component for the alignment to be correct.

Also, love the idea of using the icons here where we can ❤️, we should update across the site 😅

Reply:
Oo yeah, I'll look into it!